stack.yaml

```diff
@@ -46,7 +46,7 @@ extra-deps:
 
 # cardano-addresses-2.1.0
 - git: https://github.com/input-output-hk/cardano-addresses
-  commit: e0ab4587266430a08734e1aec1c29d261a9a3b70
+  commit: 1399751501b03346265b2e429e5832bacd6df279
```

Comment:
We should only pick released version of cardano-addresses. But I am quite reluctant releasing it in the state it is right now :/ ... I'd rather have it compliant with CIP5 for the release.

Reply:
agree, pasted it as it is just in order to have a way to interact with what we have in cardano-addresses